lib/console/messages.rb

```diff
@@ -11,6 +11,10 @@ def options
     ]
   end
 
+  def prompt() "> " end
+
+  def unrecognised() "Sorry I didn't recognise that" end
```

Comment:
I find it a bit unusual to have a method on one line like this! Probably better to have it over three lines, even though it's short.